crates/bevy_solari/src/pathtracer/pathtracer.wgsl

```diff
@@ -40,23 +40,23 @@ fn pathtrace(@builtin(global_invocation_id) global_id: vec3<u32>) {
     var p_bounce = 0.0;
     var bounce_was_perfect_reflection = true;
     var previous_normal = vec3(0.0);
-    loop {
+    for (var bounces = 0; bounces < 10; bounces++) {
```

Comment:
I think my other PR fixes this, we don't need to limit it to 10 bounces anymore. Let me know if it fixes it for you.